<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1171</width>
     <height>751</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>上 传</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设 置</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>971</width>
         <height>751</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>971</width>
           <height>751</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>4</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QGroupBox" name="groupBox">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>471</width>
               <height>271</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="title">
              <string>基础设置</string>
             </property>
             <widget class="QPushButton" name="choose">
              <property name="geometry">
               <rect>
                <x>370</x>
                <y>20</y>
                <width>91</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>浏览</string>
              </property>
             </widget>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>351</width>
                <height>41</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                   <underline>false</underline>
                  </font>
                 </property>
                 <property name="text">
                  <string>图片目录：</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::TextEditable</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="geometry">
              <rect>
               <x>490</x>
               <y>10</y>
               <width>471</width>
               <height>51</height>
              </rect>
             </property>
             <property name="title">
              <string>上传位置</string>
             </property>
             <widget class="QRadioButton" name="radioButton">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>20</y>
                <width>93</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>bilibili</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
             <widget class="QRadioButton" name="radioButton_3">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>20</y>
                <width>61</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>头条</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="radioButton_4">
              <property name="geometry">
               <rect>
                <x>190</x>
                <y>20</y>
                <width>93</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>网易严选</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="geometry">
              <rect>
               <x>490</x>
               <y>70</y>
               <width>471</width>
               <height>51</height>
              </rect>
             </property>
             <property name="title">
              <string>返回链接样式</string>
             </property>
             <widget class="QRadioButton" name="radioButton_2">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>20</y>
                <width>51</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>URL</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
             </widget>
             <widget class="QRadioButton" name="radioButton_5">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>20</y>
                <width>101</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Markdown</string>
              </property>
             </widget>
             <widget class="QRadioButton" name="radioButton_6">
              <property name="geometry">
               <rect>
                <x>190</x>
                <y>20</y>
                <width>93</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>HTML</string>
              </property>
             </widget>
            </widget>
            <widget class="QPushButton" name="Begin">
             <property name="geometry">
              <rect>
               <x>830</x>
               <y>310</y>
               <width>131</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>开始上传</string>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>580</x>
               <y>320</y>
               <width>233</width>
               <height>31</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>总数:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>成功：</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>失败：</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>330</y>
               <width>331</width>
               <height>21</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>l</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>4</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPlainTextEdit" name="plainTextEdit">
             <property name="geometry">
              <rect>
               <x>3</x>
               <y>-4</y>
               <width>961</width>
               <height>366</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
